src/cnv/cnmops/tests/test_convert_bedGraph_to_Granges.py

```diff
@@ -0,0 +1,25 @@
+import filecmp
+import subprocess
+from os.path import join as pjoin
+from . import get_resource_dir
+
+resources_dir = get_resource_dir(__file__)
+script_path = "/src/cnv/cnmops/convert_bedGraph_to_Granges.R"
```

Comment:
Is this working when running the tests inside the docker?
I thought the context in the docker is cnv.

Maybe a relative path is a better approach here